Description: Debian will include the favicon.png locally, to avoid remote access
 when reading docs (privacy breach)
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2017-10-20
Forwarded: no

Index: grads-2.2.1/doc/gadoc.html
===================================================================
--- grads-2.2.1.orig/doc/gadoc.html
+++ grads-2.2.1/doc/gadoc.html
@@ -4,7 +4,7 @@
 <title>GrADS Documentation</title>
 <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
 <link rel="stylesheet" type="text/css" href="COLA.css">
-<link rel="shortcut icon" href="http://brand.gmu.edu/wp-content/uploads/favicon.png">
+<link rel="shortcut icon" href="favicon.png">
 </head>
 
 <body>
